### PR TITLE
Basic Chromecast functionality working

### PR DESCRIPTION
**Pull request by deHank**
Hey, I'm new to using github, but I got basic chromecast functionality with subs working in the app. However certain videos with the dobly audio codec aren't working/will have no audio on some clients. 

There is also a bug where it will not get subs for certain shows, but still present the option to play them.

### CODE REVIEW

File: app/phone/build.gradle.kts [hunk 1]
```diff
@@ -110,6 +110,7 @@ dependencies {
     implementation(libs.androidx.work)
     implementation(libs.glide)
     implementation(libs.hilt.android)
+    implementation("com.google.android.gms:play-services-cast-framework:21.3.0")
```
Comment (Yash-Garg) [inline app/phone/build.gradle.kts:113]: Can move this into version catalogs in [libs.versions.toml](https://github.com/deHank/findroid/blob/chromecastMain/gradle/libs.versions.toml)

**Comment by Yash-Garg** [comment]
As mentioned in https://github.com/jarnedemeulemeester/findroid/pull/139#issuecomment-1209148041, I think we can move out the chromecast module to a separate app flavor which will not be built for fdroid.

It might require an overall large refactor.

**Comment by deHank** [comment]
Hey I wanted to give an update on this, I have chromecast working now + subtitles + audio track switching.

**Comment by deHank** [comment]
Hey I wanted to reopen this after seeing I got tagged in discord. There are still some bugs/issues (like tracking of watched status not properly working)

**Pull request closed without merging**